src/views/catalog/customize/CustomizeVirtualMachine.scss

```diff
@@ -0,0 +1,37 @@
+.customize-vm {
```

Comment:
there is no need for any of this css.
you should just copy classnames from similar forms.

check http://localhost:9000/catalog/instantiate-template?template=mariadb-persistent
for example

Reply:
Oh thanks!!

Reply:
It's a little different as the buttons should be in a sticky position but it can help for sure. thanks again

Reply:
No stick things please